addons/website_event_track/static/src/js/service_worker.js

```diff
@@ -236,14 +236,21 @@ const matchCache = async (request) => {
 
 /**
  *
- * @param {FetchEvent} param0
+ * @param {Request} request
+ * @param {object} [options]
+ *      {Boolean} disableTracking whether adding a header preventing the server to track the request (default: false)
```

Comment:
I think the syntax for those should be as following:

```js
/**
  * @param {Object} [options]
  * @param {Boolean} [options.disableTracking] blabla
*/
```

Reply:
Thanks, I didn't know. It is changed.